src/watermark.js

```diff
@@ -0,0 +1,26 @@
+import {BasePlugin} from 'kaltura-player-js';
+import {WatermarkContainer} from './components/WatermarkContainer/WatermarkContainer';
+
+export class Watermark extends BasePlugin {
+    static defaultConfig = {
+        duration: 2,
+        text: 'watermarked',
+        showEvery: 120
+    };
+
+    static isValid() {
+        return true;
+    }
+
+    getUIComponents() {
```

Comment:
important to know that now the player supports adding (and replacing/removing) component  dynamically
by `player.ui.addComponent` api (gets the same object as `getUIComponents` )

Reply:
I know, thanks :)